chainweb.cabal

```diff
@@ -21,6 +21,17 @@ tested-with:
 extra-source-files:
     CHANGELOG.md
     README.md
+    LICENSE
+
+    -- pact test data
+    pact/coin-contract/coin.repl
+    pact/coin-contract/coin.pact
+    pact/coin-contract/coin-sig.pact
+    pact/coin-contract/load-coin-contract.yaml
+    pact/coin-contract/spv-sig.pact
```

Comment:
You could also glob these too, as you did for the golden files.

Reply:
usage of `*` requires a common suffix. So it wouldn't help that much here. Also, I still prefer an explicit list, if it's not getting too long.